### ISSUE TITLE
Skype calls and video calls don't work anymore

### ISSUE THREAD

**Issue by username02** [TheGoddessInari/hamsket#213]
### Steps to reproduce
1. Install Hamsket latest version
2. Add Skype service
3. See that Call and Video Call buttons are greyed out (with 'browser not supported' tooltip)

### Expected behavior
Skype works fine

### Actual behavior
It is not possible to call using Skype

### ENVIRONMENT

> Hamsket: 6989a23  -> 518e6b0

> OS: OsX 10.14.6

> Arch: x64

Using Hamsket d78a4ea calls and video calls work perfectly; with more recent versions they don't and it is useless even the UA edit.

**Comment by Snailboy2** [comment]
Yeah just realized that to when i updated my Hamsket client. Is there a way to download the older version?

**Comment by LiBOSS-CZ** [comment]
Yes it is. https://dev.azure.com/TheGoddessInari/hamsket/_build?definitionId=2 select desired version and then you can download published artifacts.

**Comment by TheGoddessInari** [comment]
Please let me know if you figure out which version(s) are good, and which are bad.

**Comment by username02** [comment]
> Please let me know if you figure out which version(s) are good, and which are bad.

I'm quite sure the last Skype-working version is Hamsket d78a4ea

**Comment by Snailboy2** [comment]
> 
> 
> Yes it is. https://dev.azure.com/TheGoddessInari/hamsket/_build?definitionId=2 select desired version and then you can download published artifacts.

Sweet! Thanks i found it. 


> 
> 
> > Please let me know if you figure out which version(s) are good, and which are bad.
> 
> I'm quite sure the last Skype-working version is Hamsket [d78a4ea](https://github.com/TheGoddessInari/hamsket/commit/d78a4ea394558a1a2475571ddef1f1fc769554ec)

Can confirm. Downgraded to d78a4ea and now the sound in Skype works (and now media keys control the sound in Skype again)

**Comment by TheGoddessInari** [comment]
Actively working on trying to figure this out.

**Comment by TheGoddessInari** [comment]
Apologies for the delay. 

I took the simplest test for this: I downgraded Electron (with identical Hamsket code) and it works.

It's unclear what Skype is latching onto and claiming browser not supported in this instance.
This broke between 8.x and 9.x.

I'll continue to drill down into it further, but this could just be another "What are you **doing** Electron?" sort of issue. :)

**Comment by username02** [comment]
I don't know if this [https://github.com/getferdi/ferdi/issues/973](https://github.com/getferdi/ferdi/issues/973) may be of any help but hope so :)

**Comment by TheGoddessInari** [comment]
The user agent is already set, never uses Chromium.

Directly setting the user agent as suggested simply doesn't work.

Hamsket had this functionality for a long time.

**Comment by TheGoddessInari** [comment]
As I said previously, this is simply an upstream Electron issue. I've asked for help regarding that, and have received not a single reply anywhere about it from anyone.
